packages/pharos/scripts/build-react.js

```diff
@@ -121,7 +121,12 @@ export const buildReact = async () => {
 
           ${reactInterface}
 
-          const tag = new PharosClass().localName;
+          let registeredName;
+          try {
+            registeredName = new PharosClass().localName;
+          } catch(err) {}
```

Comment:
How is this error response being used here?

Reply:
It's not 😛 In the future this could be updated to check if it's a duplicate class registration and warn folks to use the registration utility instead